index.html

```diff
@@ -75,33 +75,39 @@ <h3 style="margin-top: 0px; font-weight: bold;">Updates</h3>
   <table class="table staff-table">
     <tr>
       <td><img alt="@marcus-darden" class="avatar" src="https://avatars1.githubusercontent.com/u/1323862?v=3&amp;s=120"></td>
-      <td><img src="/static/img/cgagnon-square.jpg"  alt="Cameron Gagnon"/></td>
-      <td><img src="/static/img/samkhan-square.jpg"  alt="Samiur Khan"   /></td>
+      <td><img src="/static/img/samkhan-square.jpg"  alt="Samiur Khan" /></td>
       <td><img src="/static/img/cyanliu-square.jpg" alt="Christina Liu"/></td>
+      <td><img src="/static/img/cgagnon-square.jpg"  alt="Amrit Hariharan"/></td>
```

Comment:
Good change here. The image source for Amrit is still cgagnon though.

Reply:
Hahaha yup, it's a placeholder since I'm still waiting on Amrit's pic. Wasn't sure how to put a `FIXME` in HTML 😛